blog-service/2024-08-16-manage.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Data Deletion (Manage)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - manage data
+  - ingestion
+  - data deletion
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce **Data Deletion**, which enables you to remove sensitive or inadvertently ingested data, addressing compliance and security risks caused by misconfigurations, developer errors, or third-party sources.
+
```

Comment:
Since this is a closed beta, you should add language similar to what you used in your [Kanso beta release note](https://help.sumologic.com/release-notes-service/2024/02/23/ui/):
>To explore these improvements, reach out to your Sumo Logic sales representative for access to the Closed Beta. We look forward to your feedback as we continue to refine our UI.
>
>More information:
>[Documentation](https://help.sumologic.com/docs/get-started/sumo-logic-ui-new/)

Reply:
Good point! I'm not sure if this is meant to be GA or Beta. I will find out... I've inquired in the Asana project.

Reply:
thank you again! confirmed it's beta.